d2graph/d2graph.go

```diff
@@ -413,6 +413,11 @@ func (obj *Object) Text() *d2target.MText {
 	if obj.IsContainer() {
 		fontSize = obj.Level().LabelSize()
 	}
+	if obj.Attributes.Style.FontSize != nil {
+		if size, err := strconv.Atoi(obj.Attributes.Style.FontSize.Value); err == nil {
```

Comment:
While I agree that we should've errored out before if this was wrong, this is still a string after all and who knows what code path could lead this to panic.
If it is already valid, I wonder if we shouldn't have it properly set in `d2graph.Scalar` to use the parsed value instead of `Value` as a string everywhere and requiring to cast it every time we need to use it

Reply:
yeah, it should. that's an old TODO

Reply:
https://github.com/terrastruct/d2/issues/285